### PR TITLE
Fix bank statements

### PR DESCRIPTION
**Pull request by BackSH00TER**
**Describe Pull request**
This PR adds the functionality for the bank statements tab in the bank screen. The majority of this logic already existed and just needed a few functions to do the database reads and writes.

- Added two functions `addBankStatement` and `getBankStatements`
- Uncommented the statements tab from the html, allowing it to render
- Added an extra 10px to the max-height of the container to account for the logoff button spilling out of the container once Statement button is added
- Added two new strings for deposit/withdraw statements

Before (logoff button overflowing container)
image

After 
image

Statements tab showing deposits / withdraws
image

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes]
- Does your code fit the style guidelines? [yes]
- Does your PR fit the contribution guidelines? [yes]

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -204,16 +204,40 @@ local function format_int(number)
     return minus .. int:reverse():gsub("^,", "") .. fraction
 end
 
+-- Get all bank statements for the current player
+local function getBankStatements(cid)
+    local bankStatements = MySQL.Sync.fetchAll('SELECT * FROM bank_statements WHERE citizenid = ? ORDER BY record_id DESC LIMIT 30', { cid })
```
Comment (GhzGarage) [inline server/main.lua:209]: Change to MySQL.query.await

File: server/main.lua [hunk 2]
```diff
@@ -204,16 +204,40 @@ local function format_int(number)
...
+    return bankStatements
+end
+
+-- Adds a bank statement to the database
+local function addBankStatement(cid, accountType, amountDeposited, amountWithdrawn, accountBalance, statementDescription)
+    local time = os.date("%Y-%m-%d %H:%M:%S")
+    MySQL.Async.insert('INSERT INTO `bank_statements` (`account`, `citizenid`, `deposited`, `withdraw`, `balance`, `date`, `type`) VALUES (?, ?, ?, ?, ?, ?, ?)', {
```
Comment (GhzGarage) [inline server/main.lua:216]: Change to MySQL.insert

File: server/main.lua [hunk 3]
```diff
@@ -204,16 +204,40 @@ local function format_int(number)
...
+        accountType,
+        cid,
+        amountDeposited,
+        amountWithdrawn,
+        accountBalance,
+        time,
+        statementDescription
+    }, function(_)
```
Comment (BerkieBb) [inline server/main.lua:224]: This function can be removed completely, it is unnecessary if it doesn't execute any code

**Review by BerkieBb — CHANGES_REQUESTED** [review]

File: server/main.lua (hunk 1 above)
Comment (BackSH00TER) [reply server/main.lua]: fixed

File: server/main.lua (hunk 3 above)
Comment (BackSH00TER) [reply server/main.lua]: fixed

**Review by BerkieBb — APPROVED** [review]

**Review by iktray — APPROVED** [review]

**Comment by TransitNode** [comment]
Works fine, statements need to be added for withdraws done on atms.

**Comment by BackSH00TER** [comment]
Hi @BerkieBb and @GhzGarage apologies for the ping, is there anything you'd like change for this PR to be merged?

**Comment by GhzGarage** [comment]
> Hi @BerkieBb and @GhzGarage apologies for the ping, is there anything you'd like change for this PR to be merged?

There are conflicts with your other PR i merged

**Pull request merged**